docs/notes/bugfix-15157.md

```diff
@@ -0,0 +1 @@
+# Replace '-' with '_' in android standalone identifier
```

Comment:
@livecodepanos The markdown isn't rendering correctly. I think you need to replace `'` with back-tick otherwise the `_` is recognised as the beginning of an emphasised chunk